clients/flash/common-library/src/main/actionscript/org/bigbluebutton/lib/user/views/UsersViewBase.as

```diff
@@ -26,17 +25,18 @@ package org.bigbluebutton.lib.user.views
 			
 			_userLabel = new Label();
 			_userLabel.percentWidth = 100;
-			_userLabel.styleName = "content";
-			_userLabel.text = "Users";
+			_userLabel.styleName = "sectionTitle";
+			_userLabel.text = "Online";
 			addElement(_userLabel);
 			
 			_userList = new List();
 			_userList.percentWidth = 100;
-			var itemRendererClass:ClassFactory = new ClassFactory(getItemRendererClass());
-			_userList.itemRenderer = itemRendererClass;
+			_userList.percentHeight = 100;

```

Comment:
What happens when there's 20 users in the meeting though. Won't setting the height cause a scrollbar on the list?